spec/dummy/db/migrate/20150707155107_create_secure_meetings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSecureMeetings < ActiveRecord::Migration
+  def change
+    enable_extension 'uuid-ossp'
+
+    create_table :secure_meetings do |t|
```

Comment:
You're creating a table to use uuids in the id column right? I think this should be 
```ruby
create_table :secure_meetings, id: :uuid do |t|
```